README.md

```diff
@@ -68,6 +70,7 @@ Extension definitions MUST contain a header section that defines the following f
 * **Extension Name**: The extension's unique *Registered Name*
 * **Authors**: The names of the individuals who authored the extension
 * **Minimum OCFL Version**: The minimum OCFL version that the extension requires, eg. *1.0*
+* **Extensions Specification Version**: The version of the OCFL Extensions Specification that the extension conforms to, eg. *1.0*
```

Comment:
Is the term "Extensions Specification Version" confusing? At the moment the "specification" is titled "OCFL Community Extensions" -- I think we'll need to make these consistent and I would suggest sticking with "OCFL Community Extensions"

Reply:
so I should change the header name to: "OCFL Community Extensions Version"?

Reply:
Consistency is helpful. +1 to "OCFL Community Extensions Version"... assuming it is not appropriate to take the approach of the line above: "Minimum OCFL Community Extensions Version".

Reply:
I think "minimum" implies more about the evolution of extensions than we can guarantee. I would stick with "OCFL Community Extensions Version" for the version the extension was written against

Reply:
Thanks - updated.